### PR TITLE
First attempt at theme

### PR DESCRIPTION
**Pull request by jemus42**
Sorry for the ameteurish attempt, but I wanted a black+red theme :/  
Current issues include bad visibility for buttons, and the text on the settings menu is still black and I can't figure out how to fix it. I'm hoping someone else who'd be interested in such a theme could help improve it.

### CODE REVIEW

File: scss/themes/_offline.scss [hunk 1]
```diff
@@ -20,7 +20,8 @@ body.theme-scarlet.offline,
 body.theme-seafoam.offline,
 body.theme-gecko.offline,
 body.theme-ozark.offline,
-body.theme-cobalt.offline
+body.theme-cobalt.offline,
```
Comment (danieldiekmeier) [inline scss/themes/_offline.scss]: Oh, nice. Seems like you also fixed a missing comma here. 👍

Comment (nolanlawson) [reply scss/themes/_offline.scss]: oops yes, that's a bug

**Comment by jemus42** [comment]
Okay, name suggestions:

- Pitch
- Gloom
- Carmine

… or something relating to [this fittingly titled novel I know nothing about](https://en.wikipedia.org/wiki/The_Red_and_the_Black) maybe.

**Comment by jemus42** [comment]
Also: Regarding my aformentioned text color issue, see the settings tab:



:(

**Comment by jemus42** [comment]
Okay, I'm much happier with this now. I hope "Gloom" is whimsical enough :)

**Comment by nolanlawson** [comment]
Thanks a lot! I built the code and took a look at it, but I feel that the style doesn't quite fit with the other dark themes. I think the "Gloom" name is great though, and I encourage you to put this theme on your own fork of Pinafore.

(Also thanks for helping me find that CSS comma typo! 😊)

**Pull request closed without merging**